modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc

```diff
@@ -286,17 +248,60 @@ $ oc get cgu -n default cgu-1 -ojsonpath='{.status}' | jq
   ],
   "status": {
     "currentBatch": 1,
-    "currentBatchStartedAt": "2022-02-25T15:54:16Z",
-    "remediationPlanForBatch": {
-      "spoke1": 0,
-      "spoke2": 1
+    "currentBatchRemediationProgress": {
+       "spoke1": {
+          "policyIndex": 1,
+          "state": "InProgress"
+       },
+       "spoke2": {
+          "policyIndex": 1,
+          "state": "InProgress"
+       }
     },
+    "currentBatchStartedAt": "2022-02-25T15:54:16Z",
     "startedAt": "2022-02-25T15:54:16Z"
   }
 }
 ----
 <1> Reflects the update progress of the current batch. Run this command again to receive updated information about the progress.
 
+. Check the status of the policies by running the following command:
++
+[source,terminal]
+----
+oc get policies -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE   NAME                                        REMEDIATION ACTION    COMPLIANCE STATE     AGE
+spoke1    default.policy1-common-cluster-version-policy enforce               Compliant            18m
+spoke1    default.policy2-common-nto-sub-policy         enforce               NonCompliant         18m
+spoke1    default.policy3-common-ptp-sub-policy         inform                NonCompliant         18m
+spoke1    default.policy4-common-sriov-sub-policy       inform                NonCompliant         18m
+spoke2    default.policy1-common-cluster-version-policy enforce               Compliant            18m
+spoke2    default.policy2-common-nto-sub-policy         enforce               NonCompliant         18m
+spoke2    default.policy3-common-ptp-sub-policy         inform                NonCompliant         18m
+spoke2    default.policy4-common-sriov-sub-policy       inform                NonCompliant         18m
+spoke5    default.policy1-common-cluster-version-policy inform                NonCompliant         18m
+spoke5    default.policy2-common-nto-sub-policy         inform                NonCompliant         18m
+spoke5    default.policy3-common-ptp-sub-policy         inform                NonCompliant         18m
+spoke5    default.policy4-common-sriov-sub-policy       inform                NonCompliant         18m
+spoke6    default.policy1-common-cluster-version-policy inform                NonCompliant         18m
+spoke6    default.policy2-common-nto-sub-policy         inform                NonCompliant         18m
+spoke6    default.policy3-common-ptp-sub-policy         inform                NonCompliant         18m
+spoke6    default.policy4-common-sriov-sub-policy       inform                NonCompliant         18m
+default   policy1-common-ptp-sub-policy                 inform                Compliant            18m
+default   policy2-common-sriov-sub-policy               inform                NonCompliant         18m
+default   policy3-common-ptp-sub-policy                 inform                NonCompliant         18m
+default   policy4-common-sriov-sub-policy               inform                NonCompliant         18m
```

Comment:
Actually, without changing anything else, we could assume policy1& policy 2 are bound to spoke1 & spoke2, and policy3&policy4 are bound to spoke5&spoke6. This could reduce the output to 
```suggestion
spoke1    default.policy1-common-cluster-version-policy enforce               Compliant            18m
spoke1    default.policy2-common-nto-sub-policy         enforce               NonCompliant         18m
spoke2    default.policy1-common-cluster-version-policy enforce               Compliant            18m
spoke2    default.policy2-common-nto-sub-policy         enforce               NonCompliant         18m
spoke5    default.policy3-common-ptp-sub-policy         inform                NonCompliant         18m
spoke5    default.policy4-common-sriov-sub-policy       inform                NonCompliant         18m
spoke6    default.policy3-common-ptp-sub-policy         inform                NonCompliant         18m
spoke6    default.policy4-common-sriov-sub-policy       inform                NonCompliant         18m
default   policy1-common-ptp-sub-policy                 inform                Compliant            18m
default   policy2-common-sriov-sub-policy               inform                NonCompliant         18m
default   policy3-common-ptp-sub-policy                 inform                NonCompliant         18m
default   policy4-common-sriov-sub-policy               inform                NonCompliant         18m
```